go.mod

```diff
@@ -79,11 +79,10 @@ require (
 	github.com/muesli/reflow v0.3.0 // indirect
 	github.com/muesli/termenv v0.15.3-0.20240618155329-98d742f6907a // indirect
 	github.com/muhlemmer/gu v0.3.1 // indirect
-	github.com/opentdf/platform/lib/ocrypto v0.1.6 // indirect
+	github.com/opentdf/platform/lib/ocrypto v0.1.7 // indirect
```

Comment:
Can you please separate these go mod changes from the documentation update PR?

Reply:
yeah didn't mean to add these will revert back